package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "hathor-rpc-lib",
+  "private": true,
```

Comment:
Why is it private?

Reply:
This is the root package, which will never be published, so it's private

The actual packages that will be build are inside `packages/`